eventbridge-cloudwatch-cdk-java/README.md

```diff
@@ -0,0 +1,83 @@
+# Amazon EventBridge to Amazon CloudWatch
+
+This project contains a sample AWS CDK template to create an EventBridge Rule, as well as, a CloudWatch Logs Group. The EventBridge Rule publishes matched events to CloudWatch Logs. In this example, the rule filters for specific attributes in the event before sending to the CloudWatch Logs target.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/eventbridge-cloudwatch-cdk-java.
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+- [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+- [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/cli.html) installed and configured
+  
+## Deployment Instructions
+
+1. Clone the project to your local working directory
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+
+2. Change the working directory to this pattern's directory
+    ```
+    cd serverless-patterns/eventbridge-cloudwatch-cdk-java
```

Comment:
```suggestion
    cd eventbridge-cloudwatch-cdk-java
```

Reply:
On this one if you where to clone the repo using the command in step 1 it with the default behaviour it would clone it into a directory `serverless-patterns`. 

I can add another step to cd into `serverless-patterns` if that makes it easier.